internal/api/v1/server.go

```diff
@@ -12,6 +11,7 @@ import (
 )
 
 //go:generate oapi-codegen -config ../schemas/v1.config.yaml ../schemas/v1.yaml
+//go:generate oapi-codegen -old-config-style -package v1 -generate client -templates ../templates -o zz_generated_extensions.go ../schemas/v1.yaml
```

Comment:
Just to be clear what's going on here -- we're doing a second pass on the "client" generation target, but this time with overriding the templates used with our custom templates to generate the wrappers. `-old-config-style` is used since the "new style" strangely requires defining the templates inline in the yaml file, which I'd rather not do.

So, all of the normal stuff, including client and server impls gets generated by the first `go:generate` target, the second re-runs the codegen with custom templates to add our desired extensions for client ergonomics.